HISTORY.rst

```diff
@@ -3,6 +3,60 @@
 Release History
 ---------------
 
+2.0.0 (2023-04-09) Wellton
+++++++++++++++++++++++++++
+
+**Special notes**
+
+Mesa 2.0 includes several breaking changes that provide significant improvements to Mesa.
+
+**Breaking Changes:**
+
+* space: change `coord_iter` to return `(content,(x,y))` instead of `(content, x,y)`; this reduces known errors of scheduler to grid mismatch #1566, #1723
+* space: change NetworkGrid `get_neighbors` to `get_neighborhood`; improves performance #1542
+* space: raise exception when pos is out of bounds in `Grid.get_neighborhood` #1524
+* space: remove deprecations (#1520, #1687, #1688):
+    * `find_empty()`: convert this to `move_to_empty()`
+    * `num_agents`: removed parameter from `move_to_empty()`
+    * `position_agent()`: convert this to `place_agent`
+    * `neighbor_iter()`: convert this to `iter_neighborhood()`
+* batchrunner: remove deprecations #1627
+    * `batchrunner()` and `batchrunnerMP()`: convert these to `batch_run()`
```

Comment:
Aren't those supposed to be `BatchRunner` and `BatchRunnerMP` classes, not functions?